db/migrate/20151216095054_create_container_builds.rb

```diff
@@ -0,0 +1,22 @@
+class CreateContainerBuilds < ActiveRecord::Migration
+  def change
+    create_table :container_builds do |t|
+      t.string     :ems_ref
+      t.string     :name
+      t.timestamp  :creation_timestamp
+      t.string     :resource_version
+      t.string     :namespace
+      t.string     :service_account
+      t.string     :build_source_type
+      t.string     :binary
```

Comment:
@simon3z I understand that its the file name of a binary file. Its not 100% clear but the do write that it "must be a valid filename with no path segments". do you think its not important to save right now?